src/frontend/web/src/ratingpage/Ratingpage.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import Slider from "react-slick";
+import Layout from 'antd/lib/layout';
+import Button from 'antd/lib/button';
+import { LikeTwoTone, DislikeTwoTone, DislikeOutlined,
+        LikeOutlined } from '@ant-design/icons';
+
+import 'antd/dist/antd.css';
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import './Ratingpage.scss';
+import RatingCard from './RatingCard';
+
+class Ratingpage extends Component {
```

Comment:
We would like to have a uniform standard for our components - please change this to a function component.
Why function component? It looks nicer, easier to write, and has some nice syntactic sugars such as `useContext` and all the other `use`s the are not compatible with class components.